spring-cloud-config-client/src/main/java/org/springframework/cloud/config/environment/Environment.java

```diff
@@ -16,13 +16,13 @@
 
 package org.springframework.cloud.config.environment;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
```

Comment:
My mistake. The settings don't work so well with intellij (imports are grouped to .* by intellij) and i forgot to switch back to them. I will fix that!